README.md

```diff
@@ -427,10 +433,13 @@ const sessionEndedRequest = alexia.createSessionEndedRequest();
 const intentRequest = alexia.createIntentRequest('MyIntent');
 
 // Simulate request handling
-app.handle(launchRequest, (response) => {
+describe('built in intents', () => {
+  it('should handle LanuchRequest', () => {
+    app.handle(launchRequest, (response) => {
 
     // Test the response
     expect(response).to.be.defined;
+  });
```

Comment:
- `app.handle` is asynchronous, so you need to add `done` callback and call it once intent is handled
- lets use `intentRequest` for the testing example rather than the `launchRequest`

It should look something like:
```javascript
describe('(Intent) MyIntent', () => {
  it('should handle MyIntent', done => {

    // Handle Alexia request
    app.handle(intentRequest, response => {
      // Test the response
      expect(response).to.be.defined;
      done();
    });
  });
});
```